src/Processors/Transforms/Streaming/AggregatingHelper.cpp

```diff
@@ -9,111 +9,78 @@ namespace DB
 {
 namespace Streaming
 {
-Chunk AggregatingHelper::convertWithoutKey(AggregatedDataVariants & data, const AggregatingTransformParams & params)
+namespace
 {
-    assert(data.type == AggregatedDataVariants::Type::without_key);
-    return convertToChunk(params.aggregator.prepareBlockAndFillWithoutKey(data, params.final, false, ConvertAction::STREAMING_EMIT));
+Chunk mergeBlocksToChunk(BlocksList && blocks)
+{
+    Chunk merged_chunk;
```

Comment:
Let's do necessary reserve here for merged_chunk by looping throw all blocks to calculate the total sum rows.

Reply:
Since the first chunk will be moved over which will override what we have reserved. Let's init empty chunk columns here for merged_chunk to avoid the first move assignment.